lib/Model/RelatedResource.php

```diff
@@ -49,7 +49,7 @@ class RelatedResource implements IRelatedResource, IDeserializable, JsonSerializ
 	public static float $IMPROVE_MEDIUM_LINK = 1.3;
 	public static float $IMPROVE_HIGH_LINK = 1.8;
 	public static float $IMPROVE_OCCURRENCE = 1.4;
-	public static float $UNRELATED = 0.1;
+	public static float $UNRELATED = 0.01;
```

Comment:
would 0 work as well ? 0.01 doesn't sound strict/exact-matching, it's just a reduction of the fuzziness